spec/end-to-end-tests/tests/create.js

```diff
@@ -182,19 +182,6 @@ var test_create_add_file_twice = function (browser) {
 module.exports = {
   'Create a dataset, happy path': test_create_happy_path,
 //  'Create a dataset, invalid title': test_create_invalid_title,
-  'Create a dataset, region autocomplete': test_create_region_autocomplete,
```

Comment:
this test and some of the next deleted ones are still defined above. Are you keeping them there in case you want to run them again in the future?